README.md

```diff
@@ -0,0 +1,568 @@
+    -c <bool>
+    --color <bool>             Enable color output when output is to a terminal
+                                (default: true)
+    -C <filename>
+    --css <filename>           Link for an external CSS file.
+    -g <classname>
+    --highlighter <classname>  Use custom source code highlighting php class
+    -V
+    --version                  Print the PhD version information
+    -h
+    --help                     This help
+    -e <extension>
+    --ext <extension>          The alternative filename extension to use,
+                                including the dot. Use 'false' for no extension.
+    -S <bool>
+    --saveconfig <bool>        Save the generated config (default: false).
+
+    -Q
+    --quit                     Don't run the build. Use with --saveconfig to
+                                just save the config.
+    -k
+    --packagedir               Use an external package directory.
+```
+
+As you can see, there are plenty of options to look into in PhD. The
+most important options are those which allow you to select a format and
+package to output your documentation to.
+
+```shell
+$ phd --list
+Supported packages:
+        Generic
+                xhtml
+                bigxhtml
+                manpage
+        IDE
+                xml
+                funclist
+                json
+                php
+                phpstub
+        PEAR
+                xhtml
+                bigxhtml
+                php
+                chm
+                tocfeed
+        PHP
+                xhtml
+                bigxhtml
+                php
+                howto
+                manpage
+                pdf
+                bigpdf
+                kdevelop
+                chm
+                tocfeed
+                epub
+                enhancedchm
+```
+
+
+> **_NOTE:_**
+> The format packages are provided by separate PEAR packages
+> (doc.php.net/PhD_Generic, doc.php.net/PhD_IDE, doc.php.net/PhD_PEAR
+> and doc.php.net/PhD_PHP) where only the Generic is installed by
+> default.
+
+You can tell by the output of the `--list` option that PhD can also be
+used to render the docs as a PDF file or as Unix Man Pages.
+
+To select a format and package, you must use the `-f [formatName]` and
+`-P [packageName]` options.
+
+```shell
+$ phd -f manpage -P PHP -d .manual.xml
+```
+
+This command will output the documentation for PHP functions in the Unix
+Man page format.
+
+## Compiling the PhD guide
+
+The PhD guide is this manual you are reading. It lives in
+PhD's Git repository under `docs/phd-guide/phd-guide.xml`. If you
+installed PhD from Git, you already have it. Otherwise, get it:
+
+```shell
+$ svn checkout http://svn.php.net:/repository/phd/trunk/docs/phd-guide
+U phd/docs/phd-guide/phd-guide.xml
+```
+
+Now you have everything you need. Just type
+
+```shell
+$ cd phd/docs/phd-guide/
+$ phd -f bigxhtml -d phd-guide.xml
+```
+
+There should be an .html file in the directory now. View it with a
+browser!
```

Comment:
Possibly removed, or at least the commands need to be updated because we don't use SVN any more.

Reply:
The `phd-guide.xml` file referenced here is the file that I've copied all this from so I'll just remove this section. :-)